scalableminds/webknossos
# Disable brush in mags higher than mag8

### URL of deployed dev instance (used for testing):
- https://disablebrushinhighmags.webknossos.xyz

### Steps to test:
- open a tracing, switch to brush tool
- ensure that brush is usable in mags 1 to 8
- the brush tool should get disabled (and another tool should be activated) automatically as soon as one zooms to an unusable mag

### Issues:
- fixes #4967

------
